src/items/DownloadButtonFileItem.tsx

```diff
@@ -14,13 +14,15 @@ interface DownloadButtonFileItemProps {
   url?: string;
   id?: string;
   text?: string;
+  handleLoad?: () => void;
```

Comment:
Can we change this prop name to `onClick` ? or was there a particular reason why it is called `handleLoad` ?